guesser.rb

```diff
@@ -0,0 +1,31 @@
+play_again = true
+
+while play_again
+    puts "Think of a number from 1-100"
+    user_number = gets.chomp.to_i
```

Comment:
Sorry another thing I missed in the directions it says: 
> Of course, the program won't be able to tell right away that you are lying, as it will have to exhaust the possible answers

But we are getting the `user_number` and the program will know right away that you're lying. But instead we should exhaust all possible answers. So it's more of an honesty thing, you think of a number in your head and then you tell your program if it is correct.